packages/neutrino-preset-node/index.js

```diff
@@ -35,11 +35,13 @@ module.exports = (neutrino) => {
   neutrino.use(compile, {
     include: [neutrino.options.source, neutrino.options.tests],
     babel: {
-      plugins: [require.resolve('babel-plugin-dynamic-import-node')],
+      plugins: [[require.resolve('fast-async'), { spec: true }], require.resolve('babel-plugin-dynamic-import-node')],
```

Comment:
I don't know what value I want. What I want:

- Using async functions without generators (no requirement on babel-polyfill, transform-regenerator, etc.)
- Not having a nodent runtime in every file that uses async functions